mono/utils/mono-dl.c

```diff
@@ -143,6 +148,24 @@ get_dl_name_from_libtool (const char *libtool_file)
 	return line;
 }
 
+static const char *
+fix_libc_name (const char *name)
```

Comment:
This should be inside an #ifdef ENABLE_NETCORE.

Reply:
I think the linker should get rid of it? Updated though, can't hurt.